src/app/results/results.component.html

```diff
@@ -55,6 +55,7 @@
         </div>
         <div class="container-fluid">
           <div class="combo-box">
+              <app-infobox></app-infobox>
```

Comment:
If that doesn't work (after unsubscribing query), then you have to call this ```<app-infobox>``` with some check flag like ```<app-infobox *ngIf="resultsObtained"></app-infobox>```